tests/foreman/cli/test_variables.py

```diff
@@ -752,235 +1145,139 @@ def test_negative_create_matcher_avoid_duplicate(self):
-        """Impact on variable on overriding the variable value from attribute.
-
-        @id: 07622cab-26f9-4826-a61b-64748e4866a1
-
-        @steps:
-
-        1.  Create a variable.
-        2.  Associate variable with fqdn/hostgroup.
-        3.  From host/hostgroup, override the variable value.
-
-        @assert:
-
-        1.  The host/hostgroup is saved with changes.
-        2.  New matcher for fqdn/hostgroup created inside variable.
-
-        @caseautomation: notautomated
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier1
-    def test_negative_create_override_from_attribute(self):
-        """No impact on variable on overriding the variable
-        with invalid value from attribute.
-
-        @id: 471fd4b3-c88a-4d3d-a142-343c75803c36
-
-        @steps:
-
-        1.  Create a variable.
-        2.  Associate variable with fqdn/hostgroup.
-        3.  From host/hostgroup, Attempt to override the variable with invalid
-        key type of value.
-
-        @assert:
-
-        1.  No matcher for fqdn/hostgroup is created inside variable.
-
-        @caseautomation: notautomated
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier1
-    def test_negative_create_empty_value_matcher_from_attribute(self):
-        """Override variable value with empty value from attribute.
-
-        @id: 49dab842-b310-4149-b187-bd7ccf5464a6
-
-        @steps:
-
-        1.  Create a variable.
-        2.  Associate variable with fqdn/hostgroup.
-        3.  From host/hostgroup, Attempt to override the variable with empty
-        value.
-
-        @assert:
-
-        1.  Matcher for fqdn/hostgroup created for blank value.
-
-        @caseautomation: notautomated
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier1
-    def test_positive_update_matcher_from_attribute(self):
-        """Impact on variable on editing the variable value from attribute.
-
-        @id: aa28f59f-0327-486b-8b16-64a2fe025e38
-
-        @steps:
-
-        1.  Create a variable.
-        2.  Associate variable with fqdn/hostgroup.
-        3.  Create a matcher for fqdn/hostgroup with valid details.
-        4.  From host/hostgroup, edit the variable value.
-
-        @assert:
-
-        1.  The host/hostgroup is saved with changes.
-        2.  Matcher value in variable is updated from fqdn/hostgroup.
-
-        @caseautomation: notautomated
         """
+        smart_variable = make_variable({
+            'variable': gen_string('alpha'),
+            'puppet-class': self.puppet['name'],
+            'default-value': gen_string('numeric'),
+            'variable-type': 'integer'
+        })
+        with self.assertRaises(CLIReturnCodeError):
+            SmartVariable.update({
+                'name': smart_variable['name'],
+                'merge-overrides': 1,
+                'avoid-duplicates': 1,
+            })
 
     @run_only_on('sat')
-    @stubbed()
     @tier1
-    def test_negative_update_matcher_from_attribute(self):
```

Comment:
There is no way to interact with smart variables from host interface like we have in UI, so it is invalid case